programmers/난이도별/level02.더 맵게/Jaewon0702.py

```diff
@@ -0,0 +1,19 @@
+import heapq
+
+def solution(scoville, K):
+    heapq.heapify(scoville)
+    n=0
+    while(len(scoville)>1) :
+        state=False
+        if scoville[0]<K :
+            state=True
+            heapq.heappush(scoville,heapq.heappop(scoville)+(heapq.heappop(scoville)*2))
+            n+=1
+
+        if state == False :
+            break

```

Comment:
```suggestion
        if not scoville[0]<K : break
        heapq.heappush(scoville,heapq.heappop(scoville)+(heapq.heappop(scoville)*2))
        n+=1
```

`state`는 없어도 될 것 같습니다!